### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by lslezak**
Updated version and changes handling in CONTRIBUTING.md, discussed [here](https://lists.opensuse.org/yast-devel/2016-08/msg00033.html).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -44,25 +44,27 @@ follows:
      to the [Ruby style
      guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md).
 
-  4. Make sure your change didn't break anything by building the RPM package
+  4. Update the package version (usually by `rake version:bump`) and add
+     a new entry into the `*.changes` file. For bigger changes or changes which
```
Comment (jreidinger) [inline CONTRIBUTING.md]: I would add here suggestion to use `osc vc` for changelog change.

Comment (lslezak) [inline CONTRIBUTING.md]: Good point.

Comment (shundhammer) [inline CONTRIBUTING.md]: "add a new entry to", not "into"

Comment (mvidner) [inline CONTRIBUTING.md]: Update the package version (in package/\*.spec, usually by ...) ... into the package/*.changes file.

**Comment by mvidner** [comment]
Thankgivings behind outbetterings!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -44,25 +44,29 @@ follows:
      to the [Ruby style
      guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md).
 
-  4. Make sure your change didn't break anything by building the RPM package
+  4. Update the package version (in `packages/*.spec`, usually by
+     `rake version:bump`) and add a new entry into the `package/*.changes` file
+     (by `osc vc`).
```
Comment (mvidner) [inline CONTRIBUTING.md]: (by `osc vc package`)

**Comment by mvidner** [comment]
LGTM

**Comment by shundhammer** [comment]
This was there before this pull request, but it should be fixed before this is distributed among 80+ YaST subprojects:

In the "Bug Reports" section, "if you find a problem..." is duplicated.

**Comment by shundhammer** [comment]
Other than that, LGTM.

**Comment by shundhammer** [comment]
LGTM now

**Comment by ancorgs** [comment]
LGTM

**Pull request merged**